components/mixins/experience-event/industry-verticals/experienceevent-flight-reservation.schema.json

```diff
@@ -295,5 +295,5 @@
       "$ref": "https://ns.adobe.com/xdm/mixins/industry-verticals/experienceevent-reservation-details"
     }
```

Comment:
In this mixin, please add Aircraft Type - type of aircraft associated with the reservation 
and 
Aircraft ID - ID of the aircraft associated with the reservation.

Then in the ERD, the identity should point to Aircraft ID. NOT flight number

Reply:
Example value for aircraft ID can be Boeing 737